dictionaries/perun.definition.json

```diff
@@ -162,5 +162,45 @@
   "sp_authorize_notify_button": {
     "en": "Proceed to registration",
     "cs": "Pokračovat na registrační stránku"
+  },
+  "403_is_eligible_header": {
+    "en": "Access denied",
+    "cs": "Přístup zamítnut"
+  },
+  "403_is_eligible_default_text": {
+    "en": "Your account is not from Czech academic institution. Please log in with your account from academic institution.",
+    "cs": "Přístup ke službě byl zamítnut, protože Váš účet není z české akademické instituce. Přihlaste se, prosím, pomocí svého účtu u akademické instituce."
+  },
+  "403_is_eligible_default_button": {
+    "en": "Continue with other account",
+    "cs": "Pokračovat"
+  },
+  "403_is_eligible_default_contact": {
+    "en": "If you think you have used account which meets the criteria, and you are still prevented from logging in to the service, please contact us at",
+    "cs": "Pokud si myslíte, že tahle situace nastala neprávem, můžete nás kontaktovat na"
+  },
+  "403_is_eligible_no_value_text": {
+    "en": "Your account is not from Czech academic institution. Please log in with your account from academic institution.",
+    "cs": "Přístup ke službě byl zamítnut, protože Váš účet není z české akademické instituce. Přihlaste se, prosím, pomocí svého účtu u akademické instituce."
+  },
+  "403_is_eligible_no_value_button": {
+    "en": "Continue with other account",
+    "cs": "Pokračovat"
+  },
+  "403_is_eligible_no_value_contact": {
+    "en": "If you think you have used account which meets the criteria, and you are still prevented from logging in to the service, please contact us at",
+    "cs": "Pokud si myslíte, že tahle situace nastala neprávem, můžete nás kontaktovat na"
+  },
+  "403_is_eligible_old_value_text": {
+    "en": "Your account is not from Czech academic institution. Please log in with your account from academic institution.",
+    "cs": "Přístup ke službě byl zamítnut, protože plynula doba 12 měsíců od Vašeho posledního přihlášení účtem z české akademické instituce. Přihlaste se, prosím, pomocí svého účtu u akademické instituce."
+  },
+  "403_is_eligible_old_value_button": {
+    "en": "Continue with other account",
+    "cs": "Pokračovat"
+  },
+  "403_is_eligible_old_value_contact": {
+    "en": "If you think you have used account which meets the criteria, and you are still prevented from logging in to the service, please contact us at",
+    "cs": "Pokud si myslíte, že tahle situace nastala neprávem, můžete nás kontaktovat na"
   }
```

Comment:
These texts are almost the same, I see no added value of keeping three copies...

Reply:
For some reason, the intention was to keep it distinguished if they are not eligible because of not logging in for some time and if they have not logged in at all. 
Regarding the button texts, I agree, just wanted to give the possibility to have different texts in each scenario. If we agree on a common wording, I agree with consolidating the dictionary as much as possible.
@vyskocilpavel ?